Superlista/resources/productList_en.json

```diff
@@ -0,0 +1,1533 @@
+    "category": "Drinks"
+  },
+  {
+    "id": 12,
+    "name": "Sparkling wine",
+    "category": "Drinks"
+  },
+  {
+    "id": 13,
+    "name": "Wine",
+    "category": "Drinks"
+  },
+  {
+    "id": 14,
+    "name": "Water",
+    "category": "Drinks"
+  },
+  {
+    "id": 15,
+    "name": "Soda",
+    "category": "Drinks"
+  },
+  {
+    "id": 16,
+    "name": "Energy drink",
+    "category": "Drinks"
+  },
+  {
+    "id": 17,
+    "name": "Coconut Water",
+    "category": "Drinks"
+  },
+  {
+    "id": 18,
+    "name": "Powder refreshment",
+    "category": "Drinks"
+  },
+  {
+    "id": 19,
+    "name": "Tonic",
+    "category": "Drinks"
+  },
+  {
+    "id": 20,
+    "name": "Isotonic",
+    "category": "Drinks"
+  },
+  {
+    "id": 21,
+    "name": "Whisky",
+    "category": "Drinks"
+  },
+  {
+    "id": 22,
+    "name": "Sake",
+    "category": "Drinks"
+  },
+  {
+    "id": 23,
+    "name": "Rum",
+    "category": "Drinks"
+  },
+  {
+    "id": 24,
+    "name": "Cane liquor",
+    "category": "Drinks"
+  },
+  {
+    "id": 25,
+    "name": "Gin",
+    "category": "Drinks"
+  },
+  {
+    "id": 26,
+    "name": "Vodka",
+    "category": "Drinks"
+  },
+  {
+    "id": 27,
+    "name": "Juice",
+    "category": "Drinks"
+  },
+  {
+    "id": 28,
+    "name": "Iced tea",
+    "category": "Drinks"
+  },
+  {
+    "id": 29,
+    "name": "Soy drink",
+    "category": "Drinks"
+  },
+  {
+    "id": 30,
+    "name": "Tequila",
+    "category": "Drinks"
+  },
+  {
+    "id": 31,
+    "name": "Liqueur",
+    "category": "Drinks"
+  },
+  {
+    "id": 32,
+    "name": "Chocolate",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 33,
+    "name": "Candy",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 34,
+    "name": "Chewing gum",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 35,
+    "name": "Brown sugar candy",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 37,
+    "name": "Snacks",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 39,
+    "name": "Peanut butter",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 40,
+    "name": "Peanut",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 41,
+    "name": "Straw potato",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 42,
+    "name": "Marshmallow",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 43,
+    "name": "Beef",
+    "category": "Meats"
+  },
+  {
+    "id": 44,
+    "name": "Chicken",
+    "category": "Meats"
+  },
+  {
+    "id": 45,
+    "name": "Ground beef",
+    "category": "Meats"
+  },
+  {
+    "id": 46,
+    "name": "Sausage",
+    "category": "Meats"
+  },
+  {
+    "id": 47,
+    "name": "Bacon",
+    "category": "Meats"
+  },
+  {
+    "id": 48,
+    "name": "Fish",
+    "category": "Meats"
+  },
+  {
+    "id": 49,
+    "name": "Shrimp",
+    "category": "Meats"
+  },
+  {
+    "id": 50,
+    "name": "Pork",
+    "category": "Meats"
+  },
+  {
+    "id": 51,
+    "name": "Sheep",
+    "category": "Meats"
+  },
+  {
```

Comment:
{
    "id": 52,
    "name": "Egg Jam",
    "category": "Jams and sweets"
  },

Reply:
Não encontrei o que seria egg jam como alimento fora do brasil, por isso não acrescentei